packages/common/src/directives.ts

```diff
@@ -15,6 +15,7 @@ export const schemaPrepend = gql`
   directive @amplienceNumber(minimum: Int, maximum: Int) on FIELD_DEFINITION
   directive @amplienceList(minItems: Int, maxItems: Int) on FIELD_DEFINITION
   directive @amplienceConst(item: String, items: [String!]) on FIELD_DEFINITION
+  directive @amplienceExtension(name: String) on FIELD_DEFINITION
```

Comment:
If name is required, we can just make it required here (with an `!`) and not have the custom validation.